config/locales/en.yml

```diff
@@ -2120,15 +2120,15 @@ en:
       nc_taxable_income: North Carolina taxable income
       nc_use_tax: North Carolina use tax
       nj_minimal_essential_health_coverage_html: |-
-        <p>Most people who have health insurance have minimum essential coverage. “Minimal essential coverage” just refers to how much your plan covers. If you have any of the following, you most likely have it:</p>
+        <p>Most people who have health insurance have minimum essential coverage. "Minimal essential coverage" just refers to how much your plan covers. If you have any of the following, you most likely have it:<p>
```

Comment:
[pebble] looks like the `/` in the `</p>` needs to go back in on line 2123. I see the `/` added on line 2130, and I think that one stays.